sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -261,6 +262,43 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Creates a new ministry notification for each student when a legacy restriction is added to their account.
+   * @param notifications notifications information.
+   * @param auditUserId user that should be considered the one that is causing the changes.
+   * @param entityManager entity manager to execute in transaction.
+   */
+  async saveLegacyRestrictionAddedNotification(
+    notifications: LegacyRestrictionAddedNotification[],
+    auditUserId: number,
+    entityManager?: EntityManager,
+  ): Promise<void> {
+    const templateId = await this.notificationMessageService.getTemplateId(
+      NotificationMessageType.LegacyRestrictionAdded,
+    );
+    const notificationsToSend = notifications.map((notification) => ({
+      userId: notification.userId,
+      messageType: NotificationMessageType.LegacyRestrictionAdded,
+      messagePayload: {
+        email_address: this.gcNotifyService.ministryToAddress(),
+        template_id: templateId,
+        personalisation: {
+          givenNames: notification.firstName ?? "",
+          lastName: notification.lastName,
+          studentEmail: notification.email,
+          dob: getDateOnlyFormat(notification.birthDate),
+          "date/time": this.getDateTimeOnPSTTimeZone(),
```

Comment:
Is there a reason to create a variable name as `date/time` instead of `"dateTime"` following the same for `givenNames`, `studentEmail`.